lib/puppet/module.rb

```diff
@@ -302,6 +303,11 @@ def plugin_fact_directory
     subpath("facts.d")
   end
 
+  #@return [String]
+  def locale_directory
```

Comment:
This is good that follows the same pattern that `plugin_directory` and `plugin_fact_directory` use (and its probably not worth changing that), but I would expect the [metaprogramming](https://github.com/puppetlabs/puppet/pull/6331/files#diff-ef9407d2777b719bcec044e1e15ef5eaL160) above to create a `locales` method that returned this info. (I would also expect a call to `locale()` without argument to return this as well).